Sources/PrimerSDK/Classes/Data Models/PrimerFlowEnums.swift

```diff
@@ -81,15 +90,18 @@ internal enum PrimerInternalSessionFlow {
 
     case vault
```

Comment:
could we refactor this pattern somehow? starting to reach its limit

Reply:
On my plans, this will be removed.